packages/BlockQuote/BlockQuote.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Heading from '@tds/core-heading'
+import Box from '@tds/core-box'
+import safeRest from '@tds/shared-safe-rest'
+import { media } from '@tds/core-responsive'
+import ColoredTextProvider from '../../shared/components/ColoredTextProvider/ColoredTextProvider'
+
+const Wrapper = styled.blockquote({
```

Comment:
Would you consider moving the styled component to its own styles.jsx file?

Reply:
@sketchidea Thanks for the review!
I dont think this is necessary.  It's only one StyledComponent and separating it into its own file would not be consistent with other Core or Community components.